### PR TITLE
fix: fix examples, deps, and typos

### PR DESCRIPTION
**Pull request by Gaiejj**
# Description

This PR fixes examples, deps, and typos

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md) guide. (**required**)
- [x] My change requires a change to the documentation.
- [x] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,71 +38,129 @@
...
+  - [Dataset Integration](#dataset-integration)
+    - [Reward modeling](#reward-modeling)
+    - [RL fine-tuning](#rl-fine-tuning)
+- [Inference](#inference)
+  - [Interactive Client](#interactive-client)
+  - [Interactive Arena](#interactive-arena)
+  - [Why do we open source align-anything?](#why-do-we-open-source-align-anything)
+  - [Report Issues](#report-issues)
+- [Citation](#citation)
+- [License](#license)
```
Comment (zmsn-2077) [inline README.md:64]: 不需要这个目录

File: scripts/setup.sh [hunk 2]
```diff
@@ -25,13 +25,12 @@ set -x
...
 SCRIPT_DIR="$(cd "$(dirname "$0")" &>/dev/null && pwd)"
 ROOT_DIR="$(dirname "${SCRIPT_DIR}")"
-
 export PYTHONPATH="${ROOT_DIR}${PYTHONPATH:+:${PYTHONPATH}}"
 export LOGLEVEL="${LOGLEVEL:-WARNING}"
 
 MASTER_PORT_START=10000
 MASTER_PORT_END=65535
-MASTER_PORT="$(
+MASTER_PORT="$(e
```
Comment (zmsn-2077) [inline scripts/setup.sh:33]: typo?

File: pyproject.toml [hunk 3]
```diff
@@ -33,37 +33,16 @@ classifiers = [
...
-    "datasets",
-    "tokenizers >= 0.13.3",
     "accelerate",
     "deepspeed",
-    "numpy",
-    "scipy",
-    "sentencepiece",
-    "wandb",
-    "tensorboard",
-    "optree",
```
Comment (zmsn-2077) [inline pyproject.toml]: delete

File: scripts/setup.sh (hunk 2 above)
Comment (Gaiejj) [reply scripts/setup.sh:33]: fixed

**Review by XuyaoWang — APPROVED** [review]
LGTM.

**Comment by Kass123777** [comment]
LGTM.

**Review by Kass123777 — APPROVED** [review]

**Review by zmsn-2077 — APPROVED** [review]

**Pull request merged**